mlibrary/pride
# Removing `Pride.Core.boolNodeFactory`

# Overview
With the [removal of `Pride.FieldTree.ValueBoolean`](https://github.com/mlibrary/pride/pull/149), `Pride.Core.boolNodeFactory` was only ever called in `Pride.FieldTree.FieldBoolean`. The logic has been moved directly to `Pride.FieldTree.FieldBoolean`, allowing `Pride.Core.boolNodeFactory` to be removed.

## Testing
1. Build the repository (`npm run build`).
2. Run the tests to make sure they pass (`npm run test`).
   * Break the new/updated unit tests to make sure they're working properly.
3. Navigate to your local [Search repository](https://github.com/mlibrary/search) and apply the newly generated files:
   1. Open `./package.json` and add `#pride-core-boolnodefactory-migrate` at the end of the `pride` dependency URL:
      ```bash
      "pride": "git+https://github.com/mlibrary/pride.git#pride-core-boolnodefactory-migrate"
      ``` 
   2. Do a clean install of Search:
      ```bash
      rm -rf node_modules && rm package-lock.json && npm install
      ``` 
   3. Start Search (`npm start`) and look around [the site](http://localhost:3000/everything). Check to see if there are any console errors, and everything still works as expected.
- Make sure the PR is consistent in these browsers:
